packages/@aws-cdk/aws-rds/test/integ.cluster.ts

```diff
@@ -14,6 +15,7 @@ const params = new ClusterParameterGroup(stack, 'Params', {
 });
 params.setParameter('character_set_database', 'utf8mb4');
 
+const kmsKey = new kms.EncryptionKey(stack, 'DbSecurity');
```

Comment:
@rix0rrr I can't complete a build with unencrypted RDS do to security automation. Is this ok to modify because getting an exception to this policy is a bit of a headache.